js/grid.custom.js

```diff
@@ -417,7 +417,7 @@ $.jgrid.extend({
 								break;
 							}
 						}
-						var st = soptions.searchtitle !== null ? soptions.searchtitle : p.operandTitle;
+						var st = soptions.searchtitle != null ? soptions.searchtitle : p.operandTitle;
```

Comment:
Well, it blocks the build if we use `eqeqeq: true` and `eqnull: false` (which are the default settings). For now, we ignore jshint warnings (by using the `--force` switch in `grunt`), but that is only a temporary workaround. 

If we want to catch all falsy values, we could almost use something like this instead:

```javascript
var st = soptions.searchtitle || p.operandTitle;
```

but it's mostly a matter of taste I guess. I just wanted to raise the point about `jshint` (and coding style in general)